app/admin/service.rb

```diff
@@ -10,8 +10,8 @@
     column :name
     column :delivery_organisation
     column :department
-    column "Owner" do |service|
-      service.owner.email if service.owner
+    column "Missing context" do |s|
+      status_tag("Missing", class: "error") if [s.purpose, s.how_it_works, s.typical_users, s.start_page_url].include?('')
```

Comment:
Do you want to include services where those fields are `NULL` to? How about `.any?(&:blank?)`?